app/controllers/users_controller.rb

```diff
@@ -0,0 +1,23 @@
+class UsersController < ApplicationController
+  def welcome
```

Comment:
Do we need this action if it exists in the `SessionsController`?

Reply:
You're right, it's already in SessionsCont., so not needed in UsersCont.